packages/myst-cli/src/process/notebook.ts

```diff
@@ -21,7 +21,8 @@ function createOutputDirective(): { myst: string; id: string } {
 }
 
 function blockDivider(cell: ICell, index: number) {
-  const id = cell.metadata.id ?? `nb-cell-${index}`;
+  const id =
+    cell.metadata.id ?? `nb-cell-${index}-${computeHash(JSON.stringify(cell)).substring(0, 10)}`;
```

Comment:
This hash is pretty gross - at this point in the transform we do not have `slugs` so this is required to make the `nb-cell-0` labels unique across multiple notebooks. There could still be a collision if the cells are identical... Maybe it would be better to update these to use slugs during post-process transformation.

Reply:
I think that is probably the right way to go, is to do a full reset of all of the IDs across JATS.

I am running into problems that the sections are named the same, which breaks things.